end_effector_controller:
    type: position_controllers/JointTrajectoryController
    default: True
    joints:
      - Finger_1
      - Finger_2

body_controller:
    type: position_controllers/JointTrajectoryController
    default: True
    joints:
      - Joint_1
      - Joint_2
      - Joint_3
      - Joint_4
      - Joint_5
      - Joint_6

joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 100

# gazebo_ros_control:
#   pid_gains:
#       Joint_5:
#         p: 100
#         d: 1
#         i: 1
#         i_clamp: 1
#       Joint_6:
#         p: 100
#         d: 1
#         i: 1
#         i_clamp: 1
#       Joint_4:
#         p: 100
#         d: 1
#         i: 1
#         i_clamp: 1
#       Joint_1:
#         p: 100
#         d: 1
#         i: 1
#         i_clamp: 1
#       Joint_2:
#         p: 100
#         d: 1
#         i: 1
#         i_clamp: 1
#       Joint_3:
#         p: 100
#         d: 1
#         i: 1
#         i_clamp: 1
#       Finger_1:
#         p: 100
#         d: 1
#         i: 1
#         i_clamp: 1
#       Finger_2:
#         p: 100
#         d: 1
#         i: 1
#         i_clamp: 1
